packages/components/filters/filter-menu/src/trigger-button/trigger-button.tsx

```diff
@@ -64,7 +64,7 @@ const TriggerButton = (props: TFilterMenuTriggerButtonProps) => {
       {props.label}
       {props.appliedFilterValues &&
         props.appliedFilterValues.map((value) => (
-          <Chip key={value.key} label={value.label} />
+          <Chip key={value.value} label={value.label} />
```

Comment:
I am not crazy about `value.value`. We have this in audit log and it drives me a little crazy.

Reply:
Although I realize here it's just because we call the mapped items "value". But still, if this is being passed to the prop "key", I kinda liked it as "key" better I think.